traitsui/handler.py

```diff
@@ -543,11 +543,10 @@ class Controller ( Handler ):
 
     #-- HasTraits Method Overrides ---------------------------------------------
 
-    def __init__ ( self, model = None, **metadata ):
+    def __init__ ( self, **traits):
```

Comment:
This breaks the API. `Controller(model)` is the intended way to instantiate this.